### DISCUSSION TITLE
The Submission part has disappeared from craft CP

### DISCUSSION THREAD

**Discussion by tihoq** [solspace/craft-freeform#1220] [Questions & Answers]
Hi,
Part of the submissions disappeared from the craft CP for a certain period after deploying several project yaml files that have nothing to do with the freeform plugin.
I checked the trash and didn't find them there either.
I found the submissions in the database, the submissions are also visible in the export (csv, excel...) even the submission counter counts all the submissions, but the problem is that they are not displayed in the submission list on the submission page inside the freeform plugin.

The problem occurred on these versions:

- Craft Version: Pro 4.5.11.1
- Freeform Version: Pro 4.1.13

I have now updated craft and freeform plugin to the latest version and the problem is still there.

Current versions

- Craft Version: Pro 4.8.1
- Freeform Version: Pro 5.1.1

Any idea what could be the problem here.
Thank you

**Comment by kjmartens** [comment]
Sorry for the trouble you're experiencing @tihoq.

I have not heard of an issue like this reported yet. When you say part of the submissions disappeared for a certain period, can you elaborate on this? 🙂 

1. Is this 10 submissions? 100? 1000? Out of how many... roughly 5% of submissions? etc
2. Did this happen to submissions only from a certain date range? And if so, what does that date range appear to be?
3. Is this happening to a particular form, all forms, or some of the forms?
4. Are you seeing any errors logged in the browser console or Craft error log?

Thanks!

**Comment by tihoq** [comment]
Hi @kjmartens, thank you for answering.

This problem was reported after the deployment, but no one is sure that it actually happened then.

1. 50 submissions are missing, about 50% of submissions.
2. Yes, this has happened with submissions from a certain period. From 18.12.2023 - 15.02.2024. all submissions have disappeared from CP.
3. On all forms.
4. I don't see any errors logged in the browser console or in the Craft error log?

Thank you!

**Comment by kjmartens** [comment]
Sorry for the delay @tihoq,

Were you ever able to figure out what might have caused this? If I had to guess, the issue likely had something to do with the project's deployment. What could have happened is that someone just ported over the `freeform_submissions` database tables manually and not the entire site database. Keep in mind that Freeform submissions are somewhat of a hybrid of Craft Elements and their own data since Freeform uses its own fields. 🙂
